services/src/main/java/org/keycloak/exportimport/ExportImportManager.java

```diff
@@ -104,21 +116,38 @@ public void runImport() {
             throw new RuntimeException("Failed to run import", e);
         }
     }
-
+    
     public void runImportAtStartup(String dir) throws IOException {
+        System.setProperty(ExportImportConfig.STRATEGY, Strategy.IGNORE_EXISTING.toString());
         ExportImportConfig.setReplacePlaceholders(true);
-        ExportImportConfig.setAction("import");
+        // enables logging of what is imported
+        ExportImportConfig.setAction(ExportImportConfig.ACTION_IMPORT);
+        
+        // TODO: ideally the static setting above should be unset after this is run 
```

Comment:
@shawkins So, why not do it? Is there any obstacle?

Reply:
> So, why not do it?

It wasn't being done before, so it's not really in scope for this issue.

It can be removed as a comment if you like and converted to an issue.

Reply:
@shawkins It'd be good to do it as a follow-up to have the execution isolated without any side effects.